ftw/billboard/hooks.py

```diff
@@ -29,3 +29,16 @@ def add_catalog_indexes(portal, INDEXES):
     if len(indexables) > 0:
         logger.info("Indexing new indexes %s.", ', '.join(indexables))
         catalog.manage_reindexIndex(ids=indexables)
+
+
+def uninstalled(portal):
+    remove_catalog_indexes(portal)
+
+
+def remove_catalog_indexes(portal):
+    catalog = portal.portal_catalog
+    indexes = catalog.indexes()
+
+    for name, meta_type in INDEXES:
```

Comment:
INDEXES includes `DateIndex` which appears elsewhere

Reply:
@djowett-ftw hu?? `INDEXES` contains a tuple with name and meta type. `DateIndex` is the meta type. Or I don't understand what you mean?? And the uninstall test passes, so IMHO the right index gets removed.

Reply:
... and not all date indexes

Reply:
Ah I misinterpreted it as a list - sorry!